src/arviz_plots/plots/essplot.py

```diff
@@ -0,0 +1,432 @@
+        if mean_kwargs is not False:
+            mean_dims, mean_aes, mean_ignore = filter_aes(
+                plot_collection, aes_map, "mean", sample_dims
+            )
+            mean_ess = distribution.azstats.ess(
+                dims=mean_dims, method="mean", relative=relative, **stats_kwargs.get("mean", {})
+            )
+            print(f"\n mean_ess = {mean_ess}")
+
+            if "linestyle" not in mean_aes:
+                if backend == "matplotlib":
+                    mean_kwargs.setdefault("linestyle", "--")
+                elif backend == "bokeh":
+                    mean_kwargs.setdefault("linestyle", "dashed")
+
+            plot_collection.map(
+                line_xy,
+                "mean",
+                data=mean_ess,
+                x=x_range,
+                ignore_aes=mean_ignore,
+                **mean_kwargs,
+            )
+
+        sd_kwargs = copy(plot_kwargs.get("sd", {}))
+        if sd_kwargs is not False:
+            sd_dims, sd_aes, sd_ignore = filter_aes(plot_collection, aes_map, "sd", sample_dims)
+            sd_ess = distribution.azstats.ess(
+                dims=sd_dims, method="sd", relative=relative, **stats_kwargs.get("sd", {})
+            )
+            print(f"\n sd_ess = {sd_ess}")
+
+            if "linestyle" not in sd_aes:
+                if backend == "matplotlib":
+                    sd_kwargs.setdefault("linestyle", "--")
+                elif backend == "bokeh":
+                    sd_kwargs.setdefault("linestyle", "dashed")
+
+            plot_collection.map(
+                line_xy, "sd", data=sd_ess, ignore_aes=sd_ignore, x=x_range, **sd_kwargs
+            )
+
+    # plot minimum ess
+    min_ess_kwargs = copy(plot_kwargs.get("min_ess", {}))
+
+    if min_ess_kwargs is not False:
+        min_ess_dims, min_ess_aes, min_ess_ignore = filter_aes(
+            plot_collection, aes_map, "min_ess", sample_dims
+        )
+
+        if relative:
+            min_ess = min_ess / n_points
+
+        # for each variable of distribution, put min_ess as the value, reducing all min_ess_dims
+        min_ess_data = {}
+        for var in distribution.data_vars:
+            reduced_data = distribution[var].mean(
+                dim=[dim for dim in distribution[var].dims if dim in min_ess_dims]
+            )
+            min_ess_data[var] = xr.full_like(reduced_data, min_ess)
+
+        min_ess_dataset = xr.Dataset(min_ess_data)
+        print(f"\n min_ess = {min_ess_dataset}")
+
+        if "linestyle" not in min_ess_aes:
+            if backend == "matplotlib":
+                min_ess_kwargs.setdefault("linestyle", "--")
+            elif backend == "bokeh":
+                min_ess_kwargs.setdefault("linestyle", "dashed")
+
+        plot_collection.map(
+            line_xy,
+            "min_ess",
+            data=min_ess_dataset,
+            ignore_aes=min_ess_ignore,
+            x=x_range,
+            **min_ess_kwargs,
+        )
```

Comment:
can't you use `x=x_range, y=min_ess` 1d array of length 2 for x and a scalar for y? if both x and y are provided and da doesn't have `plot_axis` (which it doesn't) then `da` is used for looping but ignored for plotting which is what we want here.